src/app/components/JumpTo/index.tsx

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import Text from '#app/components/Text';
+import InlineLink from '#app/components/InlineLink';
+
+interface JumpToHeading {
+  id: string;
+  title: string;
+}
+
+interface JumpToProps {
+  jumpToData: {
+    model: {
+      jumpToHeadings: Array<{ heading: string }>;
+    };
+  };
+  eventTrackingData?: EventTrackingMetadata;
+}
+
+const JumpTo = ({ jumpToData, eventTrackingData }: JumpToProps) => {
```

Comment:
We should rename `jumpToData` to `jumpToHeadings` as thats the property name now in the response.

Reply:
Done!